user/src/components/RegisterCarousel/useRegistration.ts

```diff
@@ -38,72 +83,112 @@ const determineErrorStep = (
  * - 登録成功時の処理
  * - ローディング状態の管理
  */
-export const useRegistration = (
-  validateForm: (step: number) => Promise<boolean>,
-  goToStep: (step: number) => void,
-  currentStep: number,
-  onClose?: () => void
-) => {
-  const { register: authRegister, isLoading } = useAuth();
-  const [displayError, setDisplayError] = useState<string | null>(null);
+export function useRegistration(
+  validateForm: (step: number) => Promise<boolean>, // 各ステップのバリデーション関数
+  goToStep: (step: number) => void, // 指定ステップに移動する関数
+  currentStep: number, // 現在のステップ番号
+  handleSubmit: UseFormHandleSubmit<RegisterFormSchema> // フォーム送信ハンドラー
+) {
```

Comment:
他のフックと同様にconstで揃えたい感ある

```suggestion
export const useRegistration = (
  validateForm: (step: number) => Promise<boolean>, // 各ステップのバリデーション関数
  goToStep: (step: number) => void, // 指定ステップに移動する関数
  currentStep: number, // 現在のステップ番号
  handleSubmit: UseFormHandleSubmit<RegisterFormSchema> // フォーム送信ハンドラー
) => {
```